src/Psl/Option/Option.php

```diff
@@ -305,4 +306,78 @@ public function equals(mixed $other): bool
     {
         return Comparison\equal($this, $other);
     }
+
+    /**
+     * Combines two `Option` values into a single `Option` containing a tuple of the two inner values.
+     * If either of the `Option`s is `None`, the resulting `Option` will also be `None`.
+     *
+     * @note: If an element is `None`, both elements in the resulting tuple will be `None`.
```

Comment:
I don't think this is true either : it will just be none(). Not a tuple of [none(), none()].
You can probably just drop this line, cause you mention how it works the line above.

Reply:
Oof, right. I've thought with the logic of the `unzip()` function, that should return the tuple I mention (if I'm not mistaken again 😄 )